includes/class-wc-gateway-youcanpay.php

```diff
@@ -188,15 +188,12 @@ public function javascript_params() {
 		$youcanpay_params = [
 			'title'            => $this->title,
 			'key'              => $this->public_key,
-			'youcanpay'        => self::ID,
-			'is_test_mode'     => $this->is_in_test_mode(),
-			'youcanpay_locale' => WC_YouCanPay_Helper::get_supported_local( get_locale() ),
-			'checkout_url'     => get_site_url() . '?wc-ajax=checkout',
-			'is_pre_order'     => WC_YouCanPay_Order_Action_Enum::get_incomplete(),
-			'order_status'     => array(
-				'incomplete' => WC_YouCanPay_Order_Action_Enum::get_incomplete(),
-				'pre_order'  => WC_YouCanPay_Order_Action_Enum::get_pre_order(),
-			),
+			'gateway'          => self::ID,
+			'locale'           => WC_YouCanPay_Helper::get_supported_local( get_locale() ),
+			'checkout_url'     => WC_YouCanPay_Helper::get_ajax_checkout_url(),
+            'is_test_mode'     => $this->is_in_test_mode(),
+			'is_pre_order'     => 0,
```

Comment:
indent the array

Reply:
https://github.com/NextmediaMa/youcan-pay-for-woocommerce/pull/11/commits/bed2f80778af4957fea82ef435ba424548b56d3a